### ISSUE TITLE
All pages complete for relaunch

### ISSUE THREAD

**Issue by thisisdano** [GSA/digital.gov-resources#43]
This epic prepares us for relaunch by completing all the necessary pages — and all the necessary site functionality — required to give our readers a digital.gov experience that matches the expectations set by the existing site.

This epic tracks all the pages, components, and QA checks we'll need to complete before going live.

### Layouts

- [ ] docs
- [x] post
- [ ] event
- [x] collection

#### layout: collection
- [ ] home
- [ ] events home
- [ ] digitalgov university
- [ ] video library
- [ ] communities home
- [ ] services home
- [ ] resources home
- [ ] category page
- [ ] subcategory page
- [ ] tag page

#### layout: event
- [ ] event

#### layout: post
- [x] post
- [ ] recaps

#### layout: docs
- [ ] community page
- [ ] manage your listserv
- [ ] service page
- [ ] resource
- [ ] about home
- [ ] join home
- [ ] subscribe
- [ ] contact us
- [ ] site policies

#### External Sites
_These can all be dealt with at a later point_
- [ ] Open Opportunities
- [ ] DigitalGov Search
- [ ] U.S. Digital Registry
- [ ] DigitalGov Summit

### Components
_Number of pages that use this component in parentheses._

#### header
- [x] social bar _— `all pages`_
- [ ] search bar _— `all pages`_
- [x] top nav _— `all pages`_
- [x] categories nav _— `posts`_
- [ ] ~~subcategories nav~~ _— `posts`_

#### footer
- [x] upcoming events _— `homepage, + events `_
- [x] ~~recent posts footer~~ _— `Not needed for launch`_
- [x] ~~twitter activity~~ _— `Not needed for launch`_

#### sidebar
- [x] ~~top ten posts~~ _— `Not needed for launch`_
- [x] ~~related communities~~ (7) _— `Not needed for launch`_

#### promos / cards

- [ ] card.img-title (1)
- [ ] card.img-title-dek (1)
- [ ] card.title-date-time (1)
- [ ] card.img-title-longDescription (1)
- [ ] card.img-postMeta-longDescription (1)
- [ ] card.img-title-dek-date-time-location (1)
- [ ] card.img-title-dek-date-time (1)

#### events
- [ ] event filter (1)
- [ ] event pagination (1)
- [ ] event gcal add (1)
- [ ] event details
- [ ] embedded map 
- [ ] embedded youtube video (1)
- [ ] ~~event export~~ _— `Not needed for launch`_
- [ ] ~~event search~~ _— `Not needed for launch`_
- [ ] ~~event display picker~~ _— `Not needed for launch`_

#### posts
- [x] post.tags
- [x] post.prevNext
- [x] post.related _— `Not needed for launch`_
- [x] ~~post.comments~~ _— `Not possible right now :/`_
- [x] ~~post.addComment~~ _— `Not possible right now :/`_

#### docs
- [ ] in-page navigation

#### other
- [ ] slider (1)
- [ ] slider contents (1)
- [ ] image.2c-right (1)
- [ ] submit.subscribe (1)
- [ ] button (12)
- [ ] display picker (3)
- [ ] post.meta (3)
- [ ] search resources (1)

### QA
- [ ] Site builds on Federalist
- [ ] Pages are all usable on mobile
- [ ] Email subscribe/unsubscribe works properly
- [ ] Images and links display properly
- [ ] Embedded videos display properly
- [ ] Updating data updates pages properly

**Comment by thisisdano** [comment]
@jeremyzilar Use this issue for tracking design/build/qa issue related to pages and site functionality here. Add pages, components, or qa items if you notice anything missing!

**Comment by jeremyzilar** [comment]
It will be interesting to see how many of these pages can simply be covered by `layout: docs` or `layout: post`

Is there anyway to indicate which will be covered by what? Should this move to the Wiki?

**Comment by aalikaram** [comment]
🛑DEPRECATED🛑 - Repo no longer being maintained #43
